src/model.jl

```diff
@@ -291,12 +291,17 @@ function _add_constraint_load_shed!(
         bus_demand[sets.load_bus_idx[i], j],
     )
     if demand_flexibility.enabled
+        flexible_load_bus_idx = indexin(sets.flexible_bus_idx, sets.load_bus_idx)
+        flexible_load_map = sparse(
+                flexible_load_bus_idx, 1:num_flexible_bus, 1, sets.num_load_bus, num_flexible_bus
+            )::SparseMatrixCSC
+        #
```

Comment:
Were you intending to leave a comment here? If not, this comment can probably be removed.